### PR TITLE
Add .stateId, .fromProperties, .getProps for all versions

### PR DESCRIPTION
**Pull request by extremeheat**

### CODE REVIEW

**Comment by extremeheat** [comment]
needs https://github.com/PrismarineJS/node-minecraft-data/pull/154

File: index.js [hunk 1]
```diff
@@ -2,6 +2,24 @@ module.exports = loader
 module.exports.testedVersions = ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.4', '1.17.1', '1.18.1', 'bedrock_1.17.10', 'bedrock_1.18.0']
 
 const nbt = require('prismarine-nbt')
+const mcData = require('minecraft-data')
+const legacyPcBlocksByName = Object.entries(mcData.legacy.pc.blocks).reduce((obj, [idmeta, name]) => {
```
Comment (extremeheat) [inline index.js:6]: maybe this should go to mc-data?

**Comment by rom1504** [comment]
Can you explain the purpose?
Old versions don't have state ids

**Comment by rom1504** [comment]
Ok but we probably want to put this directly in Minecraft data in the future

**Pull request merged**